node/bft/src/helpers/cache.rs

```diff
@@ -39,6 +39,10 @@ pub struct Cache<N: Network> {
     /// The ordered timestamp map of peer IPs and their cache hits on transmission requests.
     seen_outbound_transmissions: RwLock<BTreeMap<i64, HashMap<SocketAddr, u32>>>,
     /// The map of IPs to the number of validators requests.
+    ///
+    /// Note: we don't clear this map based on time but only on peer disconnect.
+    /// This is sufficient to avoid infinite growth as the committee has a fixed number
+    /// of members.
```

Comment:
```suggestion
    /// The map of IPs to the number of validators requests
```

This shouldn't be explained within the struct member comment, but instead on the invocation of `clear_outbound_validators_requests`